documentation/public/user-guide/05-syntax-trees/04-using-ast-types/index.md

```diff
@@ -0,0 +1,16 @@
+# 5.4. Using AST Types
+
+So far we've been using the CST of a Solidity file or fragment. However, the CST is too verbose, and not always
+convenient to use for extracting information of the program. In the following example, we are going to show how to
+obtain the parameters and attributes of a function using the AST, an abstract representation of the program tree.
+
+We start as usual by parsing the input, and then we can use the `ParseOutput` root
+to create the CST type. Since it is a node of kind `FunctionDefinition`, we are using
```

Comment:
```suggestion
to create the CST type. Since it is a node of kind [`FunctionDefinition`](../../../solidity-grammar/02-definitions/08-functions/), we are using
```